dvc/remote/http.py

```diff
@@ -42,6 +42,13 @@ def __init__(self, repo, config):
         url = config.get(Config.SECTION_REMOTE_URL)
         self.path_info = self.path_cls(url) if url else None
 
+        if not self.no_traverse:
+            raise RemoteActionNotImplemented(
```

Comment:
But what does `list_cache_paths not implemented` even tell the user? We should just explain that config option `no_traverse false` is not supported by HTTP remotes.

Reply:
:thinking: , should I change the `RemoteActionNotImplemented` arguments?
It has a fixed message:

```python
        m = "{} is not supported by {} remote".format(action, scheme)
```

Reply:
Also, why raising `RemoteActionNotImplemented` if you are only setting `no_traverse`.

I'll create a new `RemoteConfigError` just for this one